hooks/conan-center.py

```diff
@@ -874,6 +875,20 @@ def test(out):
         else:
             out.warn("No 'settings' detected in your conanfile.py. Add 'settings' attribute and use 'package_id(self)' method to manage the package ID.")
 
+    @run_test("KB-H072", output)
+    def test(out):
+        def _check_conanfile_content(content, path):
+            patterns = [r'#\s*pylint\s*:\s*skip-file\s*', '#\s*pylint\s*:\s*disable-all\s*', '#\s*pylint\s*:\s*disable=']
+            for pattern in patterns:
+                if re.search(pattern, content):
+                    out.error(f"Pylint can not be skipped, remove '#pylint' line from '{path}'")
+
+        recipe_folder = os.path.dirname(conanfile_path)
+        recipes = _get_files_following_patterns(recipe_folder, ["conanfile*.py"])
```

Comment:
```suggestion
        recipes = _get_files_following_patterns(recipe_folder, ["conanfile.py", ])
```

why the `*`? Is it regex (better `\.`)?